lambda_handlers/handler.py

```diff
@@ -11,11 +13,12 @@ def handle(self, event, context):
         :param context:
         :return dict:
         """
+        print("EVENT:")
+        print(json.dumps(event))
         try:
             return self._handle(event, context)
         except Exception as e:
-            e.message = 'Bad Request: {0}'.format(e.message)
-            raise
+            raise EnvironmentError('Bad Request: {}'.format(e.message))
```

Comment:
This is done because this is how the API Gateway gets its information. I know this is wrong way normally, but we need to add "Bad Request:" to the response the API Gateway gets. Users shouldn't be getting a call stack anyway.